oauth2/handler.go

```diff
@@ -726,6 +727,37 @@ func (h *Handler) forwardError(w http.ResponseWriter, r *http.Request, err error
 	http.Redirect(w, r, urlx.CopyWithQuery(h.c.ErrorURL(), query).String(), http.StatusFound)
 }
 
+// swagger:route DELETE /oauth2/token admin deleteOAuth2Token
```

Comment:
I would prefer to not re-use the OAuth2 endpoint here but instead make it an explicit REST resource by using the plural:

```suggestion
// swagger:route DELETE /oauth2/tokens admin deleteOAuth2Token
```

What do you think?

Reply:
+1